### DISCUSSION TITLE
Parentheses missing on tutorial page

### DISCUSSION THREAD

**Discussion by Qcellaris** [inducer/pycuda#340] [Ideas]
Dear developer,

There is a tiny mistake in the tutorial at: https://documen.tician.de/pycuda/tutorial.html

In the "Executing a Kernel" and "Bonus: Abstracting Away the Complications" sections there are print statements that have missing parentheses:

`print a_doubled`
`print a`
`print a_doubled`
`print a_gpu`

I have checked the examples/demo.py which the tutorial refers to and there they are already incorporated. It's not a big deal but just wanted to inform you.

**Comment by inducer** [comment]
Thanks! Fixed in 993df4d.
